core/include/mapbox/geojsonvt_custom_tags.hpp

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include "../../../src/data/properties.h"
+
+#include <memory>
+
+namespace mapbox {
+namespace util {
+namespace geojsonvt {
+
+struct Tags {
```

Comment:
yes, Tags are just attached to a geometry - which is by default a std::map<string,string>. To reduce conversion and copies we can use our Properties instead